Humble (#5901)
ros2/ros2_documentation: source/Tutorials/Intermediate/URDF/URDF-Main.rst

URDF
====

> URDF (Unified Robot Description Format) is a file format for specifying the geometry and organization of robots in ROS. It is based on XML language. The XML specifications used in URDF are explained `here <https://wiki.ros.org/urdf/XML>`_.

Reviewer: this will fail because of https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Contributing-To-ROS-2-Documentation.html#writing-pages. how about the following?

```suggestion
URDF (Unified Robot Description Format) is a file format for specifying the geometry and organization of robots in ROS.
A URDF file follows the XML format as described on `here <https://wiki.ros.org/urdf/XML>`_.
```
